scripts/travis-pr-title-lint

```diff
@@ -138,7 +144,12 @@ function lintMessage(message) {
     throw new Error('Unable to lint an empty message.');
   }
 
-  const res = shelljs.exec(`echo ${message} | commitlint`, {silent: true});
+  const tmpFileName = tmp.tmpNameSync();
+  fs.writeFileSync(tmpFileName, message);
+
+  const res = shelljs.exec(`cat ${tmpFileName} | commitlint`, {silent: true});
```

Comment:
This `cat` won't work on Windows, will it (and do we even care?)?

Consider using [`child_process.execSync`](https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options) directly, without temporary file. Like this:

```javascript
let stdout;
try {
  stdout = require('child_process').execSync('commitlint', {
    input: message,
    windowsHide: true,
    encoding: 'utf-8',
  });
} catch (e) {
  stdout = e.stdout; // How about e.stderr ...?
  if (e.status) { // Non-zero exit code
    throw stdout;
  }
}
return stdout.trim();
```

Reply:
`cat` does actually work on travis windows workers, because the travis job runs using git bash shell (See the Windows Build Environment section at https://blog.travis-ci.com/2018-10-11-windows-early-release):

https://travis-ci.org/mozilla/web-ext/jobs/500323517#L488

To be fair, we don't care that much of making this work on windows, as the only important thing is that it runs once on travis.

Nevertheless, I like the `child_process.execSync` strategy, as it spares us from creating a temp file at all, and so I'm going to update this PR to use that approach instead.